### DISCUSSION TITLE
Download simultaneous audios and subs

### DISCUSSION THREAD

**Discussion by Luskk-hl** [kannagi0303/yt-dlp-gui#113] [Q&A] [answered]
Hi, I'm a newbie and I wanted to know if there's any way to download multiple audios and subtitles at the same time or does it have to be one at a time.

I'm wanting to download a multilanguage/multisub video.
![Sem título](https://github.com/kannagi0303/yt-dlp-gui/assets/82573525/eca3820c-b3c3-4ba0-9c8b-9d9e7d37979f)

**Comment by kannagi0303** [comment, marked as answer]
Referring to #72, I apologize for the inconvenience, but currently, this operation cannot be performed in the GUI (Graphical User Interface).
If you have complex requirements, I suggest using the command-line mode with yt-dlp.
You can use the ``--list-formats`` option to display the format numbers and then specify the desired format to download using the ``--format`` command.
For example, you can use the ``--format 1+2+3+4`` to indicate a combination of formats to download.

**Reply by Luskk-hl** [reply]
thank you so much for answering me so fast :)
